x-pack/plugins/data_visualizer/public/application/common/components/document_count_content/document_count_chart/document_count_chart.tsx

```diff
@@ -127,7 +126,9 @@ export const DocumentCountChart: FC<Props> = ({
           id="bottom"
           position={Position.Bottom}
           showOverlappingTicks={true}
-          tickFormat={dateFormatter}
+          tickFormat={(value) => xAxisFormatter.convert(value)}
+          timeAxisLayerCount={useLegacyTimeAxis ? 0 : 2}
+          style={useLegacyTimeAxis ? {} : MULTILAYER_TIME_AXIS_STYLE}
```

Comment:
Is there a known issue with the multiline axis where it doesn't use the configured timezone `dateFormat:tz`?

![image](https://user-images.githubusercontent.com/7405507/140357718-fa6b4ed3-6130-4fa8-8c24-b3f041510800.png)

The legacy axis does use the configured timezone.

Reply:
Confirmed fixed by https://github.com/elastic/kibana/pull/117398/commits/8c78c399c47f46c9c4e61f44148f1a638f334d18